autoscheduler/scraper/serializers.py

```diff
@@ -32,7 +37,7 @@ def get_meetings(self, obj): # pylint: disable=no-self-use
         return [{
             'id': str(meeting.id),
             'days': meeting.meeting_days,
-            'start_time': meeting.start_time,
-            'end_time': meeting.end_time,
+            'start_time': format_time(meeting.start_time),
+            'end_time': format_time(meeting.end_time),
```

Comment:
So this change is just pretty-printing the time?

Reply:
Basically, since before this returned a time object, which would then be transformed by the view into the format `HH:MM:SS`. Instead of relying on this behavior (and including seconds in the time despite the fact that they'll never be needed), I decided to convert time to a string in the serializer